network-api/networkapi/buyersguide/pagemodels/products/base.py

```diff
@@ -0,0 +1,370 @@
+        max_length=5000,
+        blank=True
+    )
+
+    # How it handles privacy
+
+    share_data = models.BooleanField(  # TO BE REMOVED?
+        null=True,
+        help_text='Does the maker share data with other companies?',
+    )
+
+    share_data_helptext = models.TextField(  # TO BE REMOVED?
+        max_length=5000,
+        blank=True
+    )
+
+    how_does_it_share = models.CharField(
+        max_length=5000,
+        help_text='How does this product handle data?',
+        blank=True
+    )
+
+    user_friendly_privacy_policy = ExtendedYesNoField(
+        help_text='Does this product have a user-friendly privacy policy?'
+    )
+
+    user_friendly_privacy_policy_helptext = models.TextField(
+        max_length=5000,
+        blank=True
+    )
+
+    """
+    privacy_policy_links =  one to many, defined in PrivacyPolicyLink
+    """
+
+    worst_case = models.CharField(
+        max_length=5000,
+        help_text="What's the worst thing that could happen by using this product?",
+        blank=True,
+    )
+
+    # How to contact the company
+
+    phone_number = models.CharField(
+        max_length=100,
+        help_text='Phone Number',
+        blank=True,
+    )
+
+    live_chat = models.CharField(
+        max_length=100,
+        help_text='Live Chat',
+        blank=True,
+    )
+
+    email = models.CharField(
+        max_length=100,
+        help_text='Email',
+        blank=True,
+    )
+
+    twitter = models.CharField(
+        max_length=100,
+        help_text='Twitter username',
+        blank=True,
+    )
+
+    updates = models.ManyToManyField(
+        'buyersguide.Update',
+        related_name='pniproduct',
+        blank=True
+    )
+
+    # comments are not a model field, but are "injected" on the product page instead
+
+    related_products = models.ManyToManyField(
+        'self',
+        related_name='related_pniproduct',
+        blank=True,
+        symmetrical=False
+    )
+
+    # List of fields to show in admin to hide the image/cloudinary_image field. There's probably a better way to do
+    # this using `_meta.get_fields()`. To be refactored in the future.
+    panels = product_panels
+
+    @property
+    def is_current(self):
+        d = self.review_date
+        review = datetime(d.year, d.month, d.day)
+        cutoff = datetime(2019, 6, 1)
+        return cutoff < review
+
+    @property
+    def votes(self):
+        return get_product_vote_information(self)
+
+    def to_dict(self):
+        """
+        Rather than rendering products based on the instance object,
+        we serialize the product to a dictionary, and instead render
+        the template based on that.
+
+        NOTE: if you add indirect fields like Foreign/ParentalKey or
+              @property definitions, those needs to be added!
+        """
+        model_dict = model_to_dict(self)
+        model_dict['product_type'] = self.product_type
```

Comment:
this doesn't seem to take. Should this be `BaseProduct.product_type` and then overwritten in General/Software?